src/main/java/duke/task/TaskType.java

```diff
@@ -0,0 +1,29 @@
+package duke.task;
+
+/**
+ * Represents the type of the task.
+ */
+public enum TaskType {
```

Comment:
Good work storing the task types as enums!

Reply:
Thank you!